query/config_traversal.go

```diff
@@ -11,44 +9,50 @@ import (
 	"github.com/google/cel-go/common/types/ref"
 )
 
-func TraverseConfig(ctx context.Context, id, relationType string) *models.ConfigItem {
-	var configItem models.ConfigItem
-
-	relationTypeList := strings.Split(relationType, "/")
+func TraverseConfig(ctx context.Context, id, relationType, direction string) []models.ConfigItem {
```

Comment:
I think we should still support the traversal path, there might be cases with multiple incoming items have Kustomization parents,